kyuubi-server/src/main/scala/org/apache/kyuubi/server/trino/api/v1/StatementResource.scala

```diff
@@ -54,8 +55,21 @@ private[v1] class StatementResource extends ApiRequestContext with Logging {
   @GET
   @Path("/")
   @Consumes(Array(MediaType.TEXT_PLAIN))
-  def query(statement: String, @Context headers: HttpHeaders): QueryResults = {
-    throw new UnsupportedOperationException
+  def query(statement: String, @Context headers: HttpHeaders): Response = {
+    trinoContext = TrinoContext(headers)
+    val qr = new QueryResults(
+      "id",
+      URI.create("info"),
+      URI.create("cancel"),
+      URI.create("next"),
+      null,
+      null,
+      null,
+      null,
+      null,
+      "update_type",
+      0)
+    TrinoContext.buildTrinoResponse(qr, trinoContext)
```

Comment:
it still can not work after this pr. I think keeping throw a exception is better